internal/bminventory/inventory.go

```diff
@@ -1956,6 +1956,14 @@ func (b *bareMetalInventory) updateClusterData(ctx context.Context, cluster *com
 		}
 	}
 
+	if params.ClusterUpdateParams.MachineNetworkCidr != nil && common.IsSingleNodeCluster(cluster) {
+		err = network.VerifyMachineCIDR(swag.StringValue(params.ClusterUpdateParams.MachineNetworkCidr), cluster.Hosts, log)
+		if err != nil {
+			return err
+		}
+		setMachineNetworkCIDRForUpdate(updates, swag.StringValue(params.ClusterUpdateParams.MachineNetworkCidr))
```

Comment:
Please set the value for machineCidr

Reply:
what does it mean?

Reply:
The machineCidr variable has to be set because it used later in the function.
Also the VIPs need to be cleared